packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -381,6 +384,48 @@ describeWithDOM('mount', () => {
     });
   });
 
+  describeIf(is('>= 16'), 'portals', () => {
+    it('should have portals in mount debug tree', () => {
+      const containerDiv = global.document.createElement('div');
+      const Foo = () => (
+        <div>
+          {createPortal(
+            <div className="in-portal">InPortal</div>,
+            containerDiv,
+          )}
+        </div>
+      );
+
+      const wrapper = mount(<Foo />);
+      expect(wrapper.debug()).to.equal(`<Foo>
+  <div>
+    <Portal>
```

Comment:
perhaps this could be `<Portal container={…}>` - with the container display being something like `containerDiv.outerHTML`?